src/main/java/kira/KiraBot.java

```diff
@@ -0,0 +1,88 @@
+package kira;
+
+import java.util.Scanner;
+
+import kira.command.Command;
+import kira.exception.KiraException;
+import kira.storage.SaveLoad;
+import kira.storage.TaskList;
+import kira.ui.Parser;
+import kira.ui.Ui;
+
+/**
+ * KiraBot is a CLI software that helps you manage a list
+ * of task.
+ *
+ * @author  Eric Goh
+ * @version 0.1
+ * @since   2023-01-26
+ */
+public class KiraBot {
+
+    private static final Ui UI = new Ui();
+    private static final String FILEPATH = "./store.csv";
+    private TaskList taskList;
+
+    private void run() {
+        UI.startMsg();
+
+        try {
+            this.taskList = new TaskList(SaveLoad.load(FILEPATH));
+        } catch (KiraException e) {
+            UI.errMsg(e.getMessage());
+            this.taskList = new TaskList();
+        } finally {
+            listenForCommand();
+        }
+    }
+
+    private void listenForCommand() {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
You can use try-with-resource here, Java will automatically close the Scanner for you.